_posts/2017-10-05-denounce-segwit2x.md

```diff
@@ -0,0 +1,75 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+type: posts
+layout: post
+lang: en
+category: blog
+
+title: "Bitcoin.org to denounce Segwit2x"
+permalink: /en/posts/denounce-segwit2x.html
+date: 2017-10-05
+---
+On 2017-10-11 at noon (UTC), Bitcoin.org is planning to publish a banner on every page of the site warning users about the risks of using services that will default to the so-called Segwit2x (S2X) contentious hard fork. S2X companies will be called out by name. To ensure that we only warn users against companies that will actually put user deposits at risk, we urge all companies to publicly clarify their stance before the above date, either by a highly-visible public statement or by commenting on Bitcoin.org issue #[1835](https://github.com/bitcoin-dot-org/bitcoin.org/issues/1835) (or by doing both).
```

Comment:
2X is an altcoin, not a hardfork.

Reply:
I think that using that language just makes it too confusing in this case. "Contentious hardfork" and "altcoin" are pretty similar.

Reply:
Perhaps "contentious hardfork attempt"?